Add use cases description for APCUPSD (#38793)
home-assistant/home-assistant.github.io: source/_integrations/apcupsd.markdown

ha_integration_type: integration
---

> The APC UPS Daemon {% term integration %} is used to integrate with UPS devices from [APC](www.apc.com) when the Network Information Server ([apcupsd](http://www.apcupsd.org/)) is configured on the APC devices. Use case: When the Notify integration is set up in Home Assistant, you can send notifications if the UPS switches to battery power, the battery level drops low, or simply to track the UPS load.

Reviewer: ```suggestion
The **APC UPS Daemon** {% term integration %} is used to integrate with UPS devices from [APC](www.apc.com) when the Network Information Server ([apcupsd](http://www.apcupsd.org/)) is configured on the APC devices. Use case: When the Notify integration is set up in Home Assistant, you can send notifications. For example, when the UPS switches to battery power, or when the battery level drops low. You can also use it to track the UPS load.
```
